lib/prompts/createProjectPrompt.ts

```diff
@@ -47,26 +67,34 @@ const createTemplateOptions = async (templateSource, githubRef) => {
     process.exit(EXIT_CODES.ERROR);
   }
 
-  if (!hasAllProperties(config[PROJECT_COMPONENT_TYPES.PROJECTS])) {
+  if (!hasAllProperties(config[PROJECT_COMPONENT_TYPES.PROJECTS]!)) {
     logger.error(i18n(`${i18nKey}.errors.missingPropertiesInConfig`));
     process.exit(EXIT_CODES.ERROR);
   }
 
-  return config[PROJECT_COMPONENT_TYPES.PROJECTS];
-};
+  return config[PROJECT_COMPONENT_TYPES.PROJECTS]!;
+}
 
-function findTemplate(projectTemplates, templateNameOrLabel) {
+function findTemplate(
+  projectTemplates: ProjectProperties[],
+  templateNameOrLabel: string
+): ProjectProperties | undefined {
   return projectTemplates.find(
     t => t.name === templateNameOrLabel || t.label === templateNameOrLabel
   );
 }
 
-const createProjectPrompt = async (
-  githubRef,
-  promptOptions = {},
+export async function createProjectPrompt(
+  githubRef: string,
+  promptOptions: {
+    name: string;
+    dest: string;
+    template: string;
+    templateSource: string;
+  },
```

Comment:
Remember to type `promptUser` in this prompt

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1288/commits/dbfad7214b180917cc226ffaee59f96b198bf5f3